CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Firefox Reality
+
+Firefox Reality welcomes contribution from everyone. Here are the guidelines if you are
+thinking of helping us:
+
+
+## Contributions
+
+Contributions to Firefox Reality or its dependencies should be made in the form of GitHub
+pull requests. Each pull request will be reviewed by a core contributor
+(someone with permission to land patches) and either landed in the main tree or
+given feedback for changes that would be required. All contributions should
+follow this format, even those from core contributors.
+
+Should you wish to work on an issue, please claim it first by commenting on
+the GitHub issue that you want to work on it. This is to prevent duplicated
+efforts from contributors on the same issue.
+
+Head over to [Good First Bugs](https://github.com/MozillaReality/FirefoxReality/labels/good%20first%20issue) to find
+good first bugs to start with. 
+
+## Pull Request Checklist
+
+- Branch from the master branch and, if needed, rebase to the current master
```

Comment:
nit: feel free to make `master` a link: [`master`](https://github.com/mozillareality/firefoxreality/tree/master) (and elsewhere in this doc too, if you'd like)

Reply:
nit: feel free to link to this doc for `rebase`: [`git rebase`](https://help.github.com/en/articles/about-git-rebase)